lib/documentLoaders/node.js

```diff
@@ -162,14 +143,22 @@ module.exports = ({
   }
 };
 
-function _request(request, options) {
-  return new Promise((resolve, reject) => {
-    request(options, (err, res, body) => {
-      if(err) {
-        reject(err);
-      } else {
-        resolve({res, body});
-      }
-    });
-  });
+async function _fetch({url, headers, strictSSL}) {
+  try {
+    const agent = new https.Agent({rejectUnauthorized: strictSSL});
+    const res = await httpClient.get(
+      url, {headers, agent, redirect: 'manual'});
+    const body = JSON.stringify(res.data, null, 2);
```

Comment:
We should not reserialize the body here. Either we should ensure that a parsed body can be passed through or we should set flags to prevent parsing. My preference is for the former.

Reply:
Looks like we can just avoid the stringify.  The result would just be checked if it's a string and parsed again.